dev-core

```diff
@@ -53,5 +54,5 @@ function onexit {
   ./bin/cli stop
 }
 
-./bin/cli start --log-level "$MESG_LOG_LEVEL"
+./bin/cli start --log-level "$MESG_LOG_LEVEL" --log-force-colors "$MESG_LOG_FORCECOLORS"
```

Comment:
is it really necessary to transform from ENV to flag? seems useless

Reply:
Yes, because not everyone likes color and debug mode by default ;)

And for them this is option to change.

Reply:
This one will always set the flag as true (the value is ignored by the cli)

should do something like that

```bash
if [[ "$MESG_LOG_FORCECOLORS" != "" ]]; then 
  forceColor="--log-force-colors"
fi

./bin/cli start --log-level "$MESG_LOG_LEVEL" $forceColor
```

Not sure this works but something like that to only put the flag when the env variable is present otherwise we don't put it.

@krhubert you almost had colors in your terminal ;)